e2e/webpack/src/webpack.test.ts

```diff
@@ -160,8 +171,9 @@ describe('Webpack Plugin', () => {
 
     runCLI(`build ${appName}`);
 
-    let output = runCommand(`node dist/${appName}/main.js`);
-    expect(output).toMatch(/Hello/);
+    expect(runCommand(`node dist/${appName}/main.js`)).toMatch(/Hello/);
```

Comment:
Should this be `expect(runCommand(`node dist/apps/${appName}/main.js`)).toMatch(/Hello/);`